tox.ini

```diff
@@ -12,9 +13,7 @@ python =
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/pipreqs
-commands = python setup.py test
-deps =
```

Comment:
why are the dependencies not necessary anymore? not a complain, just trying to understand the details of the change

Reply:
To declare the dependencies, we are using `pyproject.toml`. By setting `isolated_build = true` this will install them during the build step by `tox`.